test/recipes/95-test_external_tlsfuzzer_data/cert.json.in

```diff
@@ -0,0 +1,29 @@
+[
+    {"server_command": ["@SERVER@", "s_server", "-www",
+                 "-key", "tests/serverX509Key.pem",
+                 "-cert", "tests/serverX509Cert.pem"],
+     "environment": {"PYTHONPATH" : "."},
+     "server_hostname": "localhost",
+     "server_port": @PORT@,
+     "tests" : [
+	 {"name" : "test-tls13-certificate-verify.py",
+          "comment" : "tlsfuzzer doesn't like our status request (see #633)",
+          "exp_pass" : false,
+          "comment" : "tlsfuzzer doesn't like our set of algorithms (e.g., ed25519)",
+          "arguments" : ["-k", "tests/clientX509Key.pem",
+                         "-c", "tests/clientX509Cert.pem",
+                         "-n", "10",
+                         "-e", "check sigalgs in cert request",
+	                 "-p", "@PORT@"]},
+	 {"name" : "test-tls13-ecdsa-in-certificate-verify.py",
+          "comment" : "tlsfuzzer doesn't like our status request (see #633)",
+          "exp_pass" : false,
```

Comment:
`"exp_pass" : false,` is a mallet, while here we can:
1. use a much more precise approach (-x and -X options)
2. this test in this configuration fails not because of wrong status request but rather because the server isn't configured to ask for client certificates (the -verify or -Verify option is missing)

Reply:
In general, I'd say that `"exp_pass" : false` should only be used for scripts for features that are unimplemented

Reply:
OK, will update